backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js

```diff
@@ -0,0 +1,29 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const docClient = new dynamodb.DocumentClient();
+const { USER_TABLE } = require('../../constants');
+
+/**
+ * Creates a new user in dynamodb after they confirm their email.
+ */
+exports.cognitoPostConfirmationHandler = async (event, context, callback) => {
+    console.log('received: ', event);
+    
+    const params = {
+        Item: {
+            "pk": event.request.userAttributes.sub,
+            "name": event.request.userAttributes.name + " " + event.request.userAttributes.family_name,
+            "email": event.request.userAttributes.email,
+            "profilePicture":"",
```

Comment:
I don't think you need to put these fields yet, we can just put them once the user uploads a profile picture or lists a business

Reply:
So I know that because of the schema, we don't have to put the fields up yet, but I want to know why it isn't a better idea to do so. Wouldn't it be cleaner and more consistent to keep the fields up so every user object would be consistent? I assumed it would be easier to update attribute values that already exist than add new ones.